src/OpenTelemetry/Metrics/AggregationTemporalityAttribute.cs

```diff
@@ -1,33 +0,0 @@
-// <copyright file="AggregationTemporalityAttribute.cs" company="OpenTelemetry Authors">
-// Copyright The OpenTelemetry Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-// </copyright>
-
-using System;
-
-namespace OpenTelemetry.Metrics
-{
-    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = true)]
```

Comment:
we could retain this attribute, and switch to `MetricReaderTemporalityPreference`. But okay to add it back after 1.2 as well.